go/src/meguca/db/admin.go

```diff
@@ -11,7 +11,17 @@ import (
 )
 
 // Write moderation action to log
-func logModeration(tx *sql.Tx, e auth.ModLogEntry) error {
+func LogModeration(e auth.ModLogEntry) error {
```

Comment:
I'd prefer this not be exported out of the module. Please call logModeration from within the db module. Best to integrate this into `GetSameIPPosts()`.